Include URLSessionTask in errors.md (#562)
home-assistant/companion.home-assistant: docs/troubleshooting/errors.md

*   You are missing `default_config:` from your `configuration.yaml` file. It is possible to only add `zeroconf:` to `configuration.yaml` but adding `default_config:` will add [several useful integrations](https://www.home-assistant.io/components/default_config/) along with `zeroconf:`.

### Response status code was unacceptable: 400
This occurs when the data sent during set up does not meet Home Assistant's expectations. This most commonly occurs in two circumstances:

* When you are running a version of Home Assistant older than the minimum requirement (currently 0.104.0)
* You have unexpected characters in your device's name. While setting up the Mobile App integration, we attempt to remove non-standard characters and emoji (as of Home Assistant 0.112). However, it is worth simplifying your device name to remove such characters if you are getting this error.

> ### "URLSessionTask failed with error"

Reviewer: ```suggestion
### URLSessionTask failed with error
```

This error is usually caused by one of the two following issues:
*   You have denied local network access to the app. To solve the issue on iOS, open the Home Assistant entry in the system settings and verify that local network is enabled.

> *   You have configured an incorrect `external url` in your `configuration.yaml` file. E.g. when forwarding external port 443 to internal port 8123 in your router settings, you don't have to add a port for the external url, as port 443 is the port used for HTTPS.

Reviewer: ```suggestion
*   You have configured an incorrect external url in your instance. E.g. when forwarding external port 443 to your instance's port (normally 8123), you don't have to append a port to your URL.
```

You can configure the URL setting through either YAML or the UI.

Author: Thanks!
